index.js

```diff
@@ -23,8 +23,44 @@ function fastifyMysql (fastify, options, next) {
     db.dispose().then(() => done()).catch(done)
   })
 
+  // queryString needs to be explored to check if
+  /*
+  - It's safe to use
+  - It regards a batch
+      - If so I need to change the connectionpool configs to avoid graceful degradation based on the batch size and type of query (mods and insertions have double weight)
+  - Otherwise no issue nothing to see here move along sir
+  - And I send back the whole thing to the framework ready to execute, so I need to have a
+   */
+
+  const transactionMethod = async (queryArray) => {
```

Comment:
Hi @cogluca, I think is not necessary to redefine the `transactionMethod`. 

What about exposing only the `tx` method?